lib/alice/router/helpers.ex

```diff
@@ -14,9 +14,12 @@ defmodule Alice.Router.Helpers do
 
   Adds random tag to end of image urls to break Slack's img cache.
   """
-  @spec reply(String.t, Conn.t) :: Conn.t
-  @spec reply(Conn.t, String.t) :: Conn.t
-  def reply(resp, conn = %Conn{}) when is_binary(resp), do: reply(conn, resp)
+  @spec reply(String.t, %Conn{}) :: Conn.t
+  @spec reply(%Conn{}, String.t) :: Conn.t
+  @spec reply([String.t, ...], %Conn{}) :: Conn.t
+  @spec reply(%Conn{}, [String.t, ...]) :: Conn.t
+  def reply(resp, conn = %Conn{}), do: reply(conn, resp)
+  def reply(conn = %Conn{}, resp) when is_list(resp), do: random_reply(resp, conn)
```

Comment:
calling `random_reply(conn, resp)` might be better since it requires one less function call.